lib/functions_framework.rb

```diff
@@ -142,6 +143,30 @@ def http name = DEFAULT_TARGET, &block
       self
     end
 
+    ##
+    # Define a function that responds to Typed invocations over HTTP.
+    #
+    # You must provide a name for the function, and a block that implements the
+    # function. The block should take a single `Hash` argument which will be the
+    # JSON decoded request payload. It should return a `Hash` response which
+    # will be JSON encoded and written to the response.
```

Comment:
The block isn't taking a hash argument; it's taking an argument of the specified type. Maybe say something like:

```
The block should take a single argument representing the request payload. If a `request_type` is provided, the argument object will be of the given decoded type; otherwise, it will be a JSON hash. The  block should return a JSON hash or an object that implements `#to_json`.
```